osmomath/decimal_test.go

```diff
@@ -1701,3 +1701,25 @@ func (s *decimalTestSuite) TestQuoTruncate_MutativeAndNonMutative() {
 		})
 	}
 }
+
+func (s *decimalTestSuite) TestTruncateSDKIntMutAndMut() {
```

Comment:
I love this test case! 
In addition to this test case, can we add unit test for each of the methods being added?s